lib/Traits/TimeVaryingTraits.ts

```diff
@@ -1,7 +1,12 @@
-import ModelTraits from "./ModelTraits";
 import primitiveTrait from "./primitiveTrait";
+import mixTraits from "./mixTraits";
+import CatalogMemberTraits from "./CatalogMemberTraits";
+import MappableTraits from "./MappableTraits";
 
-export default class TimeVaryingTraits extends ModelTraits {
+export default class TimeVaryingTraits extends mixTraits(
+  CatalogMemberTraits,
+  MappableTraits
```

Comment:
I'm not sure something with TimeVaryingTraits will always be Mappable

Reply:
This is because `DiscretelyTimeVaryingMixin` depends on the `.show` trait to decide whether to show/hide a chart item in chart panel (similar to the behavior in master). We can probably move the show trait to `CatalogMemeberTraits` which is more in line with its actual usage?!

Reply:
Ahh ok. I don't think it's a CatalogMemberTrait, since not all members are "show"able (e.g. a group). It might have to be it's own Trait class `Showable` that's mixed into DiscretelyTimeVaryingTraits & MappableTraits.